src/node.cc

```diff
@@ -564,6 +564,47 @@ const char *signo_string(int signo) {
   }
 }
 
+static std::vector<std::string> environment_flags = {
```

Comment:
`std::vector` is kinda wasteful for this. Can you change it back to `const char*[]`?  Or better, `const char* const []`.

Reply:
will do